plugins/outputs/riemann/riemann.go

```diff
@@ -24,8 +26,12 @@ var sampleConfig = `
   url = "localhost:5555"
   ## transport protocol to use either tcp or udp
   transport = "tcp"
-  ## separator to use between input name and field name in Riemann service name
+  ## separator to use between measurement name and field name in Riemann service name
   separator = " "
+  ## set measurement name as a Riemann tag instead of prepending it to the Riemann service name
+  measurement_as_tag = false
+  ## list of Riemann tags, if specified use these instead of any Telegraf tags
+  tags = ["telegraf","custom_tag"]
```

Comment:
can you explain the use-case of this? why do you want to completely overwrite all telegraf tags?

Many tags in telegraf are used to differentiate metrics that are otherwise identical, so using this option has the potential to ruin many collected metrics. For example, cpu metrics typically look like this:

```
cpu,cpu=cpu0 usage=99
cpu,cpu=cpu1 usage=88
```

if the user then used this option, all they would see in Riemann is:

```
cpu,telegraf=custom_tag usage=88
```

which is basically a useless metric because there is no context on which cpu it has collected, and you've also lost the data of cpu0.